Source/Core/Common/Logging/LogManager.cpp

```diff
@@ -141,7 +136,9 @@ void LogManager::Log(LogTypes::LOG_LEVELS level, LogTypes::LOG_TYPE type,
 #ifdef ANDROID
 	__android_log_write(ANDROID_LOG_INFO, "Dolphinemu", msg.c_str());
```

Comment:
@Sonicadvance1 this android logging code looks very hacky. Would it not be better to create a proper log listener, or put this #ifdef in the console log listener?